kotlin-ring-ui/src/main/kotlin/ringui/Table.ext.kt

```diff
@@ -0,0 +1,124 @@
+package ringui
+
+import kotlinext.js.ReadonlyArray
+import kotlinext.js.jso
+import react.*
+import react.dom.div
+
+external interface JsIterator<T> {
```

Comment:
It would be preferable to remove `Table` from this PR and discuss it before separate `Table` PR